SurveyTests/Sources/Mocks/Submission/SubmissionStorageUseCaseMock.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  SubmissionStorageUseCaseMock.swift
+//  SurveyTests
+//
+//  Created by Khanh on 08/12/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+
+@testable import Survey
+
+final class SubmissionStorageUseCaseMock: SubmissionStorageUseCaseProtocol {
+
+    var storeCalled = false
+
+    var loadCalled = false
+    var loadReturnValue = QuestionSubmission.dummy
```

Comment:
Is this supposed to be Observable too? 
```suggestion
    var loadReturnValue = Observable.just(QuestionSubmission.dummy)
```

Reply:
No, it's not :D.